src/main/drivers/dshot.c

```diff
@@ -139,6 +141,17 @@ uint16_t getDshotTelemetry(uint8_t index)
     return dshotTelemetryState.motorState[index].telemetryValue;
 }
 
+uint16_t erpmToRpm(uint16_t erpm)
+{
+    //  rpm = (erpm * 100) / (motorConfig()->motorPoleCount / 2)
+    return (uint16_t)((uint32_t)erpm * 200) / motorConfig()->motorPoleCount;
```

Comment:
Why are we casting to `uint32_t` here? The bottleneck is the 16 bit `erpm` function parameter anyways, so casting it to 32 bit seems unnecessary.

Reply:
I see, I will fix it, I wanted to make explicit to the compiler to make erpm * 200 with uin32_t to avoid overflow, but yes, it is unnecessary. Literal 200 is already int so there won't be overflow.

Reply:
Reworked